app/views/profiles/search.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="wrapper">
+	<h1 id="search-results">Your Matches!</h1>  
+	  <ul class="list-0">
+			<% @results.each do |result| %>
+			<li>
+				<%= result.name %>
+				<% end %>
+			</li>
```

Comment:
This looks like you're generating slightly wrong HTML here, without closing the bullets